### ISSUE TITLE
Push Assignee and other details to Notion-Board

### ISSUE THREAD

**Issue by DavidMc2021** [Souvikns/Notion-Board#57]
### Need

_In order to **see who is assigned to each issue**
_As a_ **User that has setup the integration**
_I want to_ **Push the assignee information to the integration so that I can sort/filter**

### Acceptance Criteria

**Scenario:** When a GitHub issue is updated it should push the information to the Notion-board integration
**GIVEN** a successful integration
**WHEN** any update happens
**THEN** the corresponding issue ID is updated in Notion

### Tasks
- [ ] Push Assignee information
- [ ] Consider pushing "Project, Milestone, Development" as well

**Comment by Souvikns** [comment]
Assignee, Project, Milestone, and Development supports multiple entries in GitHub, In notion, we have `multi-select` would this be helpful if I sync only the names using multi-select so something like this -

Assignees: `Souvik`, `DavidMc2021` 
Projects: `NameOfSomeProject`
Milestones: `NameofSomeMilestone`

@DavidMc2021 does this work for you? I think by doing this you can then make different views in notion according to filtering certain properties.

**Comment by DavidMc2021** [comment]
Yes I think this works well the way that you planned it out. Having different views in Notion is a clever method to review the DB's.

**Comment by DavidMc2021** [comment]
Another point, if we can push "Labels" from GitHub into Notion as a :Multi-Select then we can filter based on Labels in Notion which is very handy.
